competition_inputs/BottleJSONReader.py

```diff
@@ -0,0 +1,18 @@
+import json
+
+# G J 2-7-2023
+# Opens and imports JSON File 'BottleJSON.json' as a list
+f = open("competition_inputs/BottleJSON.json")
+data = json.load(f)
+
+# Defines Array of Bottles from the list from the JSON file
+Bottles = [
+    data["Bottle1"][0],
+    data["Bottle2"][0],
+    data["Bottle3"][0],
+    data["Bottle4"][0],
+    data["Bottle5"][0],
+]
+
+# Closes the Json File
+f.close()
```

Comment:
Make sure you have installed pre-commit and are passing mypy and pylint as described in the ReadMe. Your code needs to pass all pre-commit checks in order to be merged.